EIPS/eip-867.md

```diff
@@ -0,0 +1,192 @@
+## Preamble
+
+    EIP: 867
+    Title: Standardized Ethereum Recovery Proposals
+    Author: Dan Phifer (dp@musiconomi.com), James Levy (james@taptrust.com), Reuben Youngblom (reuben@taptrust.com)
+    Type: Meta
+    Status: Draft
+    Created: 2018-02-02
+    Replaces: 156
+
+## Simple Summary
+Provide a standardized format for Ethereum Recovery Proposals (ERPs), which relate to recovery of certain classes of lost funds.  This EIP does not advocate for or against the acceptance of any particular recovery proposals.
+
+## Abstract
+This proposal identifies a common solution method that can be used to address certain classes of lost funds on the Ethereum blockchain.  In particular, it is intended to address cases where there is no disagreement about the right outcome between directly affected parties, enabling timely and low-risk solutions to many issues that have already occurred or are likely to occur again as Ethereum grows.  
+
+The solution method is divided into three parts:
+
+1. Standards that will need to be met by any follow-on ERP in order to be considered for approval. 
+2. Recommendations for a common format for ERPs to use to specify a set of corrective actions that can be interpreted by clients. 
+3. Guidelines for client teams to implement code that can read, interpret, and apply the corrective actions at a specific block.  The set of possible corrective actions is intentionally limited to minimize risk associated with any ERP. 
```

Comment:
The ERP process is a useful solution, but this step 3 should be removed and be replaced by an alternative way to gather these funds without needing client teams or irregular state changes

Reply:
I'm not sure I can follow this suggestion. Could you expand on that how to recover funds without state changes? Let's use the Parity Multi-Sig example if you don't mind.